src/ralph/admin/mixins.py

```diff
@@ -203,6 +203,19 @@ def formfield_for_manytomany(self, db_field, request=None, **kwargs):
             return db_field.formfield(**kwargs)
         return super().formfield_for_manytomany(db_field, request, **kwargs)
 
+    def get_changelist_form(self, request, **kwargs):
```

Comment:
i mean description of bulk_edit_no_fillable option (you could btw add empty list as default value in this class and describe it there)